pipelines/tasks/hisat2.wdl

```diff
@@ -16,14 +21,18 @@ task HISAT2PE {
       --rg PL:ILLUMINA --rg PU:${sample_name} \
       --new-summary --summary-file ${output_name}.log \
       --met-file ${output_name}.hisat2.met.txt --met 5 \
-      -p 4 -S ${output_name}.sam
+      --seed 12345 \
+      -k 10 \
+      --secondary \
+      -p 4 -S ${output_name}.sam 
     samtools sort -@ 4 -O bam -o "${output_name}.bam" "${output_name}.sam" 
   }
   runtime {
     docker:"quay.io/humancellatlas/secondary-analysis-hisat2:2-2.1.0"
     memory:"5 GB"
-    disks: "local-disk 100 HDD"
+    disks: "local-disk " + disk_size + " HDD"
     cpu: "4"
+    preemptible: 5
```

Comment:
gonna be so cheap!

Reply:
Yes, i will re-run a full dataset after this merged and we can figure out how cheap it will be!